### PR TITLE
Dunfell update and 2.7.0 recipes.

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
@mender-test-bot start pipeline

**Comment by kacf** [comment]
Only the last two commits are new, the rest are cherry-picks.

**Review by oleorhagen — APPROVED** [review]

File: meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "git://github.com/mendersoftware/mender-configure-module;protocol=https;branch=1.0.x"
+
+# Tag: TODO
```
Comment (lluiscampos) [inline meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb:13]: Still `TODO` ?

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Please revert [this commit](https://github.com/mendersoftware/meta-mender/commit/346f222c25f3e8f696d3f4187eb63be8c32725e6) once we have added 1.0.0 recipe for `mender-configure`

**Comment by lluiscampos** [comment]
> Please revert [this commit](https://github.com/mendersoftware/meta-mender/commit/346f222c25f3e8f696d3f4187eb63be8c32725e6) once we have added 1.0.0 recipe for `mender-configure`

Alternatively: we can just not cherry-pick it here and then revert only in `master`.

File: meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb (hunk 1 above)
Comment (kacf) [reply meta-mender-core/recipes-mender/mender-configure/mender-configure_1.0.0.bb:13]: This is explained in the commit message. Will be fixed before the merge.

**Comment by kacf** [comment]
> > Please revert [this commit](https://github.com/mendersoftware/meta-mender/commit/346f222c25f3e8f696d3f4187eb63be8c32725e6) once we have added 1.0.0 recipe for `mender-configure`
> 
> Alternatively: we can just not cherry-pick it here and then revert only in `master`.

Ah, good catch! I will simply omit this entirely.

**Comment by kacf** [comment]
@mender-test-bot: start pipeline

**Review by lluiscampos — APPROVED** [review]

**Review by lluiscampos — APPROVED** [review]

**Comment by kacf** [comment]
The final status did not make it through for some reason, but this passed.

**Pull request merged**